src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,14 +105,15 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
-              : this.config.secondary_info === "last-triggered" &&
-                stateObj.attributes.last_triggered
-              ? html`
-                  <ha-relative-time
-                    .hass=${this.hass}
-                    .datetime=${stateObj.attributes.last_triggered}
-                  ></ha-relative-time>
-                `
+              : this.config.secondary_info === "last-triggered"
+               ? stateObj.attributes.last_triggered
+               ? html`
+                   <ha-relative-time
+                     .hass=${this.hass}
+                     .datetime=${stateObj.attributes.last_triggered}
+                   ></ha-relative-time>
+                 `
+               : this.hass.localize("ui.panel.lovelace.entities.never_triggered")
```

Comment:
You should add this to https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/translations/en.json

Reply:
ok I will, referring to this PR.

Reply:
No, in this PR

Reply:
see https://github.com/home-assistant/home-assistant-polymer/pull/4444

Reply:
It should be in this PR, not a separate one

Reply:
ok, I will, if I can find how to do that... how can I add a second file to this PR to edit?

Reply:
Just edit it in the same branch: https://github.com/Mariusthvdb/home-assistant-polymer/tree/patch-3

You would also need to fix the lint issues (running prettier would fix that for you)